src/app/copy/copy-base.ts

```diff
@@ -19,18 +26,33 @@ export abstract class CopyBase {
   /**
    * A tooltip that describes the value to be copied
    */
-  @Input('tooltip') tooltip: string;
+  @Input('tooltipText') tooltipText: string;
 
   /**
    * Placement for the tooltip
    */
-  @Input('tooltipPlacement') tooltipPlacement: string = 'top';
+  @Input('tooltipPlacement') tooltipPlacement: string = this._copyDefaults.tooltipPlacement;
+
+  /**
+   * A tooltip that describes what the copy button does
+   */
+  @Input('copyBtnTooltipText') copyBtnTooltipText: string = this._copyDefaults.copyBtnTooltipText;
+
+  /**
+   * Placement for the copy button tooltip
+   */
+  @Input('copyBtnTooltipPlacement') copyBtnTooltipPlacement: string = this._copyDefaults.copyBtnTooltipPlacement;
 
   /**
    * The value to be copied to the clipboard
    */
   @Input('value') value: string;
 
+  /**
+   * A width to set on the copy container
+   */
+  @Input('width') width: string;
```

Comment:
We're basically not setting a default width so the inline copy remains inline flow content out of the box.

Block copy is encased within a div which is block level by default so it will occupy 100% of available width in the parent element.

Because the width is a shared property between inline and block, not setting a default value here seemed to make most sense. I could set `width: 100%` on `.pfng-block-copy` if you think this will become a tripping hazard for devs.

Reply:
That is fine. I was more curious if setting width was a requirement